descriptions/ab+/en_us.lua

```diff
@@ -21,473 +21,473 @@ EID.descriptions[languageCode].fonts = {{name="default"}, {name="inverted"}, {na
+	{"220", "Sad Bombs", "{{Bomb}} +5 Bombs#Isaac's bombs shoot 10 tears in a circle when they explode"},
 	{"221", "Rubber Cement", "Isaac's tears bounce off enemies and obstacles"},
-	{"222", "Anti-Gravity", "Holding the fire buttons causes tears to hover in midair#Releasing the fire buttons shoots them in the direction they were fired#↑ {{Tears}} -2 Tear Delay"},
-	{"223", "Pyromaniac", "Getting hit by explosions heals 1 Red Heart instead of damaging Isaac#Grants immunity to rock wave attacks and Mom and Satan's stomp attacks#{{Bomb}} +5 Bombs"},
-	{"224", "Cricket's Body", "Tears split up in 4 on hit#Split tears deal half damage#↑ {{Tears}} +1 Tears up#↓ {{Range}} -10 Range down"},
-	{"225", "Gimpy", "{{SoulHeart}} Taking damage has a chance to spawn a Soul Heart#{{HalfHeart}} Enemies have a chance to drop a Half Red Heart on death"},
-	{"226", "Black Lotus", "↑ {{Heart}} +1 Health up#{{SoulHeart}} +1 Soul Heart#{{BlackHeart}} +1 Black Heart"},
-	{"227", "Piggy Bank", "{{Coin}} +3 coins#Taking damage spawns 1-2 coins#{{Player14}} 0-1 coin as Keeper"},
-	{"228", "Mom's Perfume", "{{Fear}} 15% chance to shoot Fear tears#↑ {{Tears}} +1 Tears up"},
-	{"229", "Monstro's Lung", "{{Chargeable}} Tears are charged and released in a shotgun style attack#↓ {{Tears}} 4.3x Tear Delay up"},
-	{"230", "Abaddon", "↑ {{Damage}} +1.5 Damage up#↑ {{Speed}} +0.2 Speed up#{{Fear}} Fear tears#↓ {{EmptyHeart}} Removes all heart containers#{{BlackHeart}} +6 Black Hearts"},
-	{"231", "Ball of Tar", "{{Slow}} 10% chance to shoot slowing tears#{{Luck}} 100% chance at 18 Luck#Isaac leaves a trail of slowing creep"},
-	{"232", "Stop Watch", "{{Slow}} Taking damage slows all enemies in the room permanently#↑ +0.3 Speed up"},
-	{"233", "Tiny Planet", "Isaac's tears orbit around him#↑ {{Range}} Range way up#↑ +7 Tear Height#Spectral tears"},
+	{"222", "Anti-Gravity", "↑ {{Tears}} -2 Tear delay#Holding the fire buttons causes tears to hover in midair#Releasing the fire buttons shoots them in the direction they were fired"},
+	{"223", "Pyromaniac", "{{Bomb}} +5 Bombs#{{Heart}} Getting hit by explosions heals 1 heart instead of damaging Isaac#Immune to rock wave attacks and Mom and Satan's stomp attacks"},
+	{"224", "Cricket's Body", "↑ {{Tears}} -1 Tear delay#↑ {{Tearsize}} x1.2 Tear size#↓ {{Range}} -10 Range#Tears split up in 4 on hit#Split tears deal half damage"},
+	{"225", "Gimpy", "{{SoulHeart}} Taking damage has a chance to spawn a Soul Heart#{{HalfHeart}} Enemies have a chance to drop a half Red Heart on death"},
+	{"226", "Black Lotus", "↑ {{Heart}} +1 Health#{{SoulHeart}} +1 Soul Heart#{{BlackHeart}} +1 Black Heart"},
+	{"227", "Piggy Bank", "{{Coin}} +3 Coins#{{Coin}} Taking damage spawns 1-2 coins#{{Player14}} 0-1 Coin as Keeper"},
+	{"228", "Mom's Perfume", "↑ {{Tears}} -1 Tear delay#{{Fear}} 15% chance to shoot fear tears#{{Luck}} 100% chance at 85 luck"},
```

Comment:
{{Luck}} 100% chance at 85 luck"
those look to me like useless information for the average player